src/platform_impl/ios/view.rs

```diff
@@ -519,7 +519,15 @@ pub unsafe fn create_window(
             msg_send![window, setScreen:video_mode.monitor().ui_screen()]
         }
         Some(Fullscreen::Borderless(ref monitor)) => {
-            msg_send![window, setScreen:monitor.ui_screen()]
+            let uiscreen: id = match monitor {
+                Some(ref monitor) => monitor.ui_screen() as id,
```

Comment:
Usually should use `match &monitor` instead of `Some(ref monitor)`. There's a surprising amount of `ref`s here, given that they're almost never necessary in Rust anymore.

Reply:
that's a pretty old rust iirc, so that's why it's using ref, I've just mirrored existing code.